mne/viz/brain/brain.py

```diff
@@ -0,0 +1,457 @@
+        dt_max = max
+        dt_min = min if center is None else -1 * max
+        k = 1 / (dt_max - dt_min)
+        b = 1 - k * dt_max
+        act_data = k * act_data + b
+        act_data = np.clip(act_data, 0, 1)
+
+        act_color = lut(act_data)
+
+        self._data['k'] = k
+        self._data['b'] = b
+
+        for ri, v in enumerate(self._views):
+            if self._hemi != 'split':
+                ci = 0
+            else:
+                ci = 0 if hemi == 'lh' else 1
+            renderer = self._renderers[ri][ci]
+            mesh = renderer.mesh(x=self.geo[hemi].coords[:, 0],
+                                 y=self.geo[hemi].coords[:, 1],
+                                 z=self.geo[hemi].coords[:, 2],
+                                 triangles=self.geo[hemi].faces,
+                                 color=act_color,
+                                 opacity=None)
+            self._overlays[hemi + '_' + v] = mesh
+
+        # How can we make this bit universal as well???
+        # if colorbar and not self._colorbar_added:
+        #     ColorBar(self)
+        #     self._colorbar_added = True
+
+    def show(self):
+        u"""Display widget."""
+        self._renderers[0][0].show()
+
+    def update_lut(self, min=None, mid=None, max=None):
+        u"""Update color map.
+
+        Parameters
+        ----------
+        min : float | None
+            Min value in colormap.
+        mid : float | None
+            Intermediate value in colormap (middle between min and
+            max).
+        max : float | None
+            Max value in colormap.
+        """
+        alpha = self._data['alpha']
+        center = self._data['center']
+        colormap = self._data['colormap']
+        min = self._data['min'] if min is None else min
+        mid = self._data['mid'] if mid is None else mid
+        max = self._data['max'] if max is None else max
+
+        lut = _calculate_lut(colormap, alpha=alpha, fmin=min, fmid=mid,
+                             fmax=max, center=center)
+        self._data['lut'] = lut
+        return lut
+
+    @property
+    def overlays(self):
+        return self._overlays
+
+    @property
+    def data(self):
+        u"""Data used by time viewer and color bar widgets."""
+        return self._data
+
+    @property
+    def views(self):
+        return self._views
+
+    @property
+    def hemis(self):
+        return self._hemis
+
+    def _check_hemi(self, hemi):
+        u"""Check for safe single-hemi input, returns str."""
+        if hemi is None:
+            if self._hemi not in ['lh', 'rh']:
+                raise ValueError('hemi must not be None when both '
+                                 'hemispheres are displayed')
+            else:
+                hemi = self._hemi
+        elif hemi not in ['lh', 'rh']:
+            extra = ' or None' if self._hemi in ['lh', 'rh'] else ''
+            raise ValueError('hemi must be either "lh" or "rh"' + extra)
```

Comment:
`raise ValueError('hemi must be either "lh" or "rh"' + extra + ", got " + str(hemi))`